plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -848,7 +848,7 @@ export class DeferredPersonOverrideWorker {
             'processPendingOverrides',
             async () => {
                 status.debug('👥', 'Processing pending overrides...')
-                const overridesCount = await this.processPendingOverrides()
+                const overridesCount = await this.processPendingOverrides(5000)
```

Comment:
This parameter already existed because I intended to at one point make it configurable, but was too lazy to actually do that. (It seems that I am still too lazy.)

This artificially limits throughput for the sake of predictability to an arbitrarily chosen limit of 1000 overrides/sec. Under normal conditions, we don't see this many overrides.

This could be optimized in various ways to not be so clunky, but I don't think the investment is worth it.

Reply:
> It seems that I am still too lazy.

😁